hudi-client/hudi-java-client/src/main/java/org/apache/hudi/client/HoodieJavaWriteClient.java

```diff
@@ -140,6 +140,19 @@ public boolean commit(String instantTime,
     return postWrite(result, instantTime, table);
   }
 
+  @Override
+  public List<WriteStatus> insertOverwrite(List<HoodieRecord<T>> records, String instantTime) {
```

Comment:
Why do we need this operation support in `HoodieJavaWriteClient`? Shouldn't the `SparkRDDWriteClient` support be enough for deltastreamer?

Reply:
yes , it should be enough , I will remove this change